device/sycl/src/clusterization/component_connection.hpp

```diff
@@ -0,0 +1,32 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// SYCL library include(s).
+#include "traccc/sycl/utils/queue_wrapper.hpp"
+
+// Project include(s).
+#include "traccc/clusterization/detail/sparse_ccl.hpp"
+#include "traccc/edm/cell.hpp"
+#include "traccc/edm/cluster.hpp"
+
+// Vecmem include(s).
+#include <vecmem/memory/memory_resource.hpp>
```

Comment:
You should add an explicit include for `vecmem::data::vector_view`.

Reply:
done